src/util/CMakeLists.txt

```diff
@@ -0,0 +1,55 @@
+# Copyright (c) 2023 The Bitcoin Core developers
+# Distributed under the MIT software license, see the accompanying
+# file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
+add_library(bitcoin_util STATIC EXCLUDE_FROM_ALL
+  asmap.cpp
+  bip32.cpp
+  bytevectorhash.cpp
+  check.cpp
+  error.cpp
+  fees.cpp
+  getuniquepath.cpp
+  hasher.cpp
+  message.cpp
+  moneystr.cpp
+  rbf.cpp
+  readwritefile.cpp
+  settings.cpp
+  serfloat.cpp
+  sock.cpp
+  spanparsing.cpp
+  strencodings.cpp
+  string.cpp
+  syscall_sandbox.cpp
```

Comment:
Need to add a check and conditionally set `USE_SYSCALL_SANDBOX`, same as `ENABLE_TRACING`

Reply:
> Need to add a check and conditionally set `USE_SYSCALL_SANDBOX`...

It should be a part of a commit which adds CMake's analogue of `--with-seccomp` configure option,  like https://github.com/bitcoin/bitcoin/pull/25797/commits/412a8dd879ff8b04fe5c5107b6713f73376e3eb7.

Reply:
> ... same as `ENABLE_TRACING`

... https://github.com/bitcoin/bitcoin/pull/25797/commits/f4ebea9a1475c9ac7804e8986f986cd528b6d39e